screwdriver.yaml

```
@@ -23,10 +23,12 @@ jobs:
   publish:
     requires: main
     steps:
+      - yarn-cache: yarn config set cache-folder ~/.yarn
+      - install: yarn install --frozen-lockfile
       - setup-ci: git clone https://github.com/screwdriver-cd/toolbox.git /tmp/ci
       - publish: |
           . /tmp/ci/git-ssh.sh
-          npx semantic-release
+          ./node_modules/.bin/semantic-release
```

Can we do:

```suggestion
          yarn semantic-release
```

https://github.com/yahoo/ember-gridstack/blob/9d01e3d133f3c7f0e9b52c75e725e555c4c7680d/package.json#L33